index.js

```diff
@@ -42,8 +42,10 @@ module.exports = {
         'no-unused-vars': [ 'error', { 'vars': 'all', 'args': 'none' } ],
         'object-curly-spacing': [ 'error', 'always' ],
         'semi': [ 'error', 'never' ],
+        'space-infix-ops': ["error", { "int32Hint": false }],
```

Comment:
Just a small note, we use `'` instead of `"` in this file and also there is a space before `[` or `]` :-)

Reply:
RIgtht :) done.

Reply:
And what about the spaces? :-D

Reply:
Holy moly :D we should apply this linter for itself. Fixed

Reply:
Yeah, I had exactly the same idea :-D "Inception style" :-)